examples/meter/meter.html

```diff
@@ -0,0 +1,156 @@
+
+    <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+
+    <div id="example">
+      <p>This meter changes its value randomly every 5 seconds.</p>
+      <h3 id="cpu_usage_label">Central Processing Unit(CPU) Usage</h3>
+      <div role="meter" aria-valuenow="90" aria-valuemin="0" aria-valuemax="100"
+        aria-labelledby ="cpu_usage_label">
+        <div id="fill"></div>
+      </div>
+    </div>
+    <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+  </section>
+  <section>
+      <h2 id="kbd_label">Keyboard Support</h2>
+
+      <p>No keyboard interaction needed.</p>
+  </section>
+
+  <section>
+    <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
+    <table aria-labelledby="rps_label" class="data attributes">
+      <thead>
+        <tr>
+          <th scope="col">Role</th>
+          <th scope="col">Attribute</th>
+          <th scope="col">Element</th>
+          <th scope="col">Usage</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr data-test-id="meter-role">
+          <th scope="row"><code>meter</code></th>
+          <td></td>
+          <td>
+            <code>div</code>
+          </td>
+          <td>
+            <ul>
+              <li>Identifies the element as a <code>meter</code> widget.</li>
+            </ul>
+          </td>
+        </tr>
+          <tr data-test-id="meter-aria-valuemin">
+              <td></td>
+              <th scope="row"><code>aria-valuemin="NUMBER"</code></th>
+              <td><code>div</code></td>
+              <td>
+                <ul>
+                  <li>Specifies a numeric value that is the minimum value the meter can have.</li>
+                </ul>
+              </td>
+            </tr>
+            <tr data-test-id="meter-aria-vlauemax">
+                <td></td>
+                <th scope="row"><code>aria-valuemax="NUMBER"</code></th>
+                <td><code>div</code></td>
+                <td>
+                  <ul>
+                    <li>Specifies a numeric value that is the maximum value the meter can have.</li>
+                  </ul>
+                </td>
+            </tr>
+            <tr data-test-id="meter-aria-valuenow">
+                <td></td>
+                <th scope="row"><code>aria-valuenow="NUMBER"</code></th>
+                <td><code>div</code></td>
+                <td>
+                  <ul>
+                    <li>A numeric value that is the current value of the meter</li>
+                  </ul>
+                </td>
+              </tr>
+              <tr data-test-id="meter-aria-labelledby">
+                  <td></td>
+                  <th scope="row"><code>aria-labelledby</code></th>
+                  <td><code>div</code></td>
+                  <td>
+                    <ul>
+                      <li>Identifies the accessible name for <code>meter</code> widget.</li>
```

Comment:
Add "the", i.e. accessible name for the meter widget.

Reply:
Fixed in 9bcfe10